draft-ietf-quic-tls.md

```diff
@@ -1108,6 +1108,16 @@ and reliance on them should be minimized.  However, any tampering with the
 parameters will cause the handshake to fail.
 
 
+## Removing the EndOfEarlyData Message {#remove-eoed}
+
+Negotiating the quic_transport_parameters extension also modifies TLS to remove
```

Comment:
this makes it seem like sending quic_transport_params on a tcp connection will also negotiate having no eoed. Is that really desirable. Maybe we should say that this should be only done when  both endpoints know they are speaking quic.